draft-ietf-tls-tlsflags.xml

```diff
@@ -131,12 +131,13 @@
             <t> A server that supports this extension and also supports at least one of the flag-type features
                 that use this extension and that were declared by the ClientHello extension SHALL send this
                 extension with the intersection of the flags it supports with the flags declared by the client.
-                The intersection operation MAY be implemented as a bitwise AND. The server may need to send two
-                tls_flags extensions, one in the ServerHello and the other in the EncryptedExtensions message.
+                The intersection operation MAY be implemented as a bitwise AND. The server may need to send more than one
+                tls_flags extensions, up to one in each of the messages that may carry extensions: ServerHello, 
+                EncryptedExtensions, Certificate, and NewSessionTicket.
                 It is up to the document for the specific feature to determine whether support should be
-                acknowledged in the ServerHello or the EncryptedExtensions message.</t>
+                acknowledged in the ServerHello, in the EncryptedExtensions, or in one of the other messages.</t>
```

Comment:
This should probably just read: 

"It is up to the document for the specific feature to select which of these messages should acknowledge support for the feature."